vercel-llm-api/src/proxy/proxy.py

```diff
@@ -0,0 +1,29 @@
+import requests, random, re
+
+def fetch_proxy_list(protocol):
+    response = requests.get(f'https://raw.githubusercontent.com/TheSpeedX/SOCKS-List/master/{protocol}.txt')

```

Comment:
This file should not exist. It should be up to the user to figure out how to get the proxies.

Reply:
Besides, I've experimented with using those kinds of free proxy lists, and they tend to be extremely slow and unreliable.